manual/spotify/tools/ChocolateyUnInstall.ps1

```diff
@@ -0,0 +1,6 @@
+$packageName = "spotify"
+$fileType = "exe"
+$args = "--uninstall -s"
+$filePath = "$env:APPDATA\Spotify\Spotify.exe"
+ 
+Uninstall-ChocolateyPackage $packageName $fileType $args $filePath
```

Comment:
There should be a empty line at the end of every file
see [2.2 Formatting](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting)